server/routes/journeys/referral/description/schemas.ts

```diff
@@ -1,18 +1,22 @@
 import z from 'zod'
 import { Request } from 'express'
 import { createSchema } from '../../../../middleware/validationMiddleware'
+import { getMaxCharsAndThreshold } from '../../../../utils/appendFieldUtils'
 
-const DESCRIPTION_OF_CONCERN_TOO_LONG_MSG = 'Description must be 4,000 characters or less'
+const DESCRIPTION_OF_CONCERN_TOO_LONG_MSG = (length: number) =>
+  `Description must be ${length.toLocaleString()} characters or less`
 
 export const schemaFactory = async (req: Request) => {
+  const { maxLengthChars } = getMaxCharsAndThreshold(req, req.journeyData.csipRecord!.referral.descriptionOfConcern)
```

Comment:
`[Node] TypeError: Cannot read properties of undefined (reading 'referral')` in MaR journey